.github/workflows/ci.yml

```diff
@@ -25,7 +25,9 @@ jobs:
       # Should be skipped as the mint step is expected to fail
       - name: Check
         if: ${{ steps.mint.outcome == 'success' }}
-        run: echo '${{ steps.mint.outputs.api-token }}'
+        run: >-
+          echo '${{ steps.mint.outputs.api-token }}'
+          && exit 1
```

Comment:
The `fail` job is supposed to fail and it does fail in the `Mint Token` step.
But it should not fail the whole workflow as this is intentional that's why there's `continue-on-error: true`.
The `Check` step is skipped due to `if: ${{ steps.mint.outcome == 'success' }}`.
Observe this [workflow run](https://github.com/tschm/token-mint-action/actions/runs/5864037954/job/15898456609) as an example.

So, this change is not needed.

Reply:
That's the successful outcome. But if the step incorrectly succeeds, the job will still succeed, hiding the invalid behavior.

Reply:
> That's the successful outcome. But if the step incorrectly succeeds, the job will still succeed, hiding the invalid behavior.

Right. Yes, that's a plausible scenario. :thinking: 
In that case, propagating failure with `exit 1` makes sense.

Could you please make it consistent with the rest of the workflow by using `|` instead of `>-` in `run` and remove `&&`?